recipes/sleap-io/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sleap-io" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sleap_io-{{ version }}.tar.gz
+  sha256: 56e46e3a5926644412484931b35366fca3425039d97d31bd7b593e43302ae766
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+  run:
+    - python >=3.7
+    - ffmpeg <6.1.0
```

Comment:
ffmpeg is pretty stable, why do you pin on this? can this be relaxed to `ffmpeg <7.0a`?

Reply:
Hi @hmaarrfk,

Should've dropped in a comment in the source -- our pin was originally due to this upstream issue: https://github.com/imageio/imageio-ffmpeg/issues/99

I haven't had a chance to confirm whether it's fixed in `imageio`/`imageio-ffmpeg` and whether we need to lower bound that dep.

Either way, since we're not requiring `opencv` as a workaround for the above compilation issues, this is mainly a compatibility release to get on conda-forge and unblock @niksirbi on his `movement` package.

We'll sort out all the deps in a subsequent release.

(Thanks for the `opencv` feedstock btw, [longtime `opencv-python` fan](https://github.com/opencv/opencv-python/issues/36).)

Reply:
basically, these less than constraints will kill any kind of hopes of compatibility you may have.

Please test to see if you can remove this.

`av` is not compatible with ffmpeg7, i've been building both ffmpeg6 and ffmpeg7 for that reason, but generally that isn't really sustainable.

you may want to drop `av` as a requirement since it seems you support multiple backend.